ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+
+
+prosody__deploy_state: "present"
+
+
+# Packages and installation [[[
+# -----------------------------
+
+# .. envvar:: apache__base_packages [[[
+#
+# List of base packages to install.
+prosody__base_packages:
+  - prosody
+  - lua-zlib
+  - prosody-modules # for ldap auth
+#  - lua-event # for use_libevent = true
```

Comment:
Since the role files are supposed to be read-only, having commented out parameters like this is counter-productive. Instead, I would create a condition that enables a package installation if specific parameter is enabled, and skips the package otherwise. For example:
```yaml
prosody__base_packages:
  - '{{ "lua-event" if prosody__use_libevent|bool else [] }}'
```
This requires the use of the `with_flattened` parameter lookup in the APT installation task to work properly.

Reply:
thanks, done